lxd/container_lxc.go

```diff
@@ -1728,7 +1728,11 @@ func (c *containerLXC) initLXC(config bool) error {
 	}
 
 	// Setup shmounts
-	err = lxcSetConfigItem(cc, "lxc.mount.entry", fmt.Sprintf("%s dev/.lxd-mounts none bind,create=dir 0 0", c.ShmountsPath()))
+	if lxc.HasApiExtension("mount_injection") || util.RuntimeLiblxcVersionAtLeast(3, 1, 0) {
```

Comment:
Is this call to HasApiExtension actually safe?

That is, if I do:
 - Build LXD using a recent enough lxc-dev that HasApiExtension exists
 - Run the resulting binary on a system that has a liblxc which predates the introduction of HasApiExtension

Will that blow up because of go-lxc attempting to access an unknown symbol, or do we have logic in go-lxc to do a version check and avoid hitting that case?

Reply:
@brauner are you going to send a go-lxc pull request to make HasApiExtension safe to call in all cases?

Reply:
Yep, done https://github.com/lxc/go-lxc/pull/112